reactioncommerce/reaction
# ref: Update Catalog Schema

Resolves #4283  
Impact: **minor**  
Type: **refactor**

## Issue
The catalog simple schema is out of date.

## Solution
Update the existing catalog Simple Schema to match how we currently save Catalog data to the `Catalog` collection

## Breaking changes
Updates to the existing Catalog schema - although as far as I know, we have put the `Catalog` out as "use at your own risk", so I don't know if this is technically a traditional "breaking change".


## Testing
1. Start the app, make sure there are no errors
1. Make various changes to a product and publish to the catalog, make sure that there are no schema errors
